File: cards-service/src/main/java/a/gleb/cardservice/model/CardResponse.java
package a.gleb.cardservice.model;

import lombok.*;

import java.time.LocalDateTime;
import java.util.UUID;

@Getter
@Setter
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class CardResponse {

    private UUID cardId;
    private String cardTitle;
    private String description;
    private UUID userId;
    private LocalDateTime updatedTime;
}

File: card-service/src/main/kotlin/a/gleb/cardservice/model/CardResponse.kt
package a.gleb.cardservice.model

import io.swagger.v3.oas.annotations.media.Schema
import java.time.LocalDateTime
import java.util.*

@Schema(description = "Card response model")
data class CardResponse(

    @Schema(description = "ID of card entity")
    val cardId: UUID?,

    @Schema(description = "Card title")
    val cardTitle: String,

    @Schema(description = "Card description (optional)")
    val cardDescription: String?,

    @Schema(description = "Card owned username")
    val sub: String,

    @Schema(description = "Last update time for card")
    val updateTime: LocalDateTime
)
